lib/menu.csv

```diff
@@ -0,0 +1,8 @@
+dishes,prices
```

Comment:
Creating a CSV file for the menu rather than hardcoding the menu in a hash is a good example of creating clear areas of 'responsibility'